teacher.rb

```diff
@@ -9,3 +9,7 @@ def can_use_services?
     true
   end
 end
+person = Person.new(34, 'John')
+p person.name
+teacher = Teacher.new(45, 'eng')
+p(teacher)
```

Comment:
[OPTIONAL] It would be nicer to remove those lines for a better user experience.